### ISSUE TITLE
problems running example- fesom 2.0

### ISSUE THREAD

**Issue by akb75** [FESOM/fesom2#585]
I contacted Dr. Wilken-Jon von Appen some time ago about using the fesom2 model for studying fram strait circulation. 

Hi everyone! I am trying to run the fesom2 example (listed in the getting started helper doc) for a small project I'm working on to study how different variations in salinity conditions, temperature, and ice affect the flow of Fram Strait and its effects on the Atlantic Ocean. I have also contact Dr. Qiang Wang about this as well, and I was trying to find a specific contact for help regarding these issues but the online contact documentation states the best way to do so is to open a github issue. 

I am trying to run fesom2 on my laptop (windows, windows powershell, ubuntu), and I am running into some problems. I have tried several different methods listed below to configure the model. I downloaded the model off of github using git clone, and I am able to get to the refactoring branch. 
Following this, the following line gives problems: 

bash -l ./configure.sh
bash -l ./configure.sh ubuntu
This command above produces the following error: 
PS C:\Users\anant\Documents\fram_strait_project\fesom2> bash -l ./configure.sh ubuntu
Sourcing /mnt/c/Users/anant/Documents/fram_strait_project/fesom2/env/ubuntu/shell for environment
/mnt/c/Users/anant/Documents/fram_strait_project/fesom2/env/ubuntu/shell: **line 4: $'\r': command not found**

If I run ./configure.sh ubuntu, the configure file opens on my computer, but there is nothing in the bin directory where the Parms library and model executable should be built.
 
Another method I am trying is to run using docker desktop, but the following command gives me an error. 
**docker run -it -v "$(pwd)"/fesom2:/fesom/fesom2 koldunovn/fesom2_test:refactoring2 /bin/bash
docker: invalid reference format.**

I am able to pull the image, but I am not able to get inside of the container. I am not sure how to fix the errors. 

Also, when installing the following ubuntu packages, I obtain an error that 
apt-get -y install cmake vim git libnetcdf-dev libnetcdff-dev libpmi2-pmix
E: Unable to locate package libpmi2-pmix

If someone could provide any guidance on how to proceed, I would greatly appreciate this!

**Comment by koldunovn** [comment]
I don't think that anyone run ever run FESOM on windows, only in docker container. We also don't use windows machines, so it would be hard to reproduce your errors. @dsidoren I know you run FESOM with docker on linux with students, do you have some instructions?

You mention that the purpose is to study Fram strait circulation - I don't know what configuration you have in mind, but running FESOM on a laptop with anything larger than PI mesh is more or less hopeless - it will be too slow. 

So the best way to go it to try to run FESOM on *nix machine with good amount of compute cores. Even if the problems on windows will be fixed, you will not be able to run large enough configurations.
